### DISCUSSION TITLE
How do you make the gifs longer then 2 seconds ?

### DISCUSSION THREAD

**Discussion by Triskelen** [NickeManarin/ScreenToGif#1222] [General]
can you make the gifs longer then 2 seconds ? i really can't seem to find it

**Comment by NickeManarin** [comment]
Yes, of course you can.
You can record as long as you have drive space.

What exactly is happening?

**Comment by Triskelen** [comment]
my gifs are only 2 seconds long when i create them and can't seem to find the option to make it longer

**Reply by NickeManarin** [reply]
I'll need more details.

It's recording for only two seconds or is it exporting only two seconds?

1) If it's recording for only two seconds, aren't you pressing by accident a key shortcut to stop the recording?

2) If it's exporting just two seconds, aren't you using any option for partial export?

**Comment by Triskelen** [comment]
When I press record it records the whole action then the viewer opens and I can see the whole thing when I press safe the gif saves and only shows the first 2 seconds every time
